containers/Containerfile

```diff
@@ -0,0 +1,82 @@
+FROM registry.fedoraproject.org/fedora:34
+
+MAINTAINER "Candlepin Team" <candlepin@redhat.com>
+# should be the UID of whatever user is running podman, change this with: podman build --build-arg UID="$(id -u)"
+ARG UID=1000
+ARG GIT_HASH=main
+ENV SMDEV_CONTAINER_OFF='True'
+
+RUN dnf install --disablerepo=updates -y \
```

Comment:
out of curiosity: why are the updates disabled? I'd say that having up-to-date packages including fixes should help not running into issues.

Reply:
I do not recall and can't find a reason for it at the moment. If we'd like to keep it enabled that works for me.

Reply:
OK, it actually failed an hour ago because of the conflicts between new packages installed and the old non-updated packages of the container. Hence, I enabled testing, and make sure to dist-upgrade everything.